.github/workflows/default-prechecks-unix.yml

```diff
@@ -26,11 +24,6 @@ jobs:
 
       - name: Install flake8
         run: pip install flake8
-
```

Comment:
Not in `alpha` branch here. Since all the changes in this branch will be in the `unix-alpha` directory, I have removed this step of lint check. I should also mention that for the same reason, I have also disabled the workflow to release the main machine stats (.github/workflows/pypi-upload.yml file in this PR). We will use the same workflow from `master` branch only.

To conclude,
add a release tag on master branch > release new version of the main machine stats
add a release tag on alpha branch > release new version of the machine stats alpha